tools/stress-cluster/scripts/find_all_stress_packages.ps1

```diff
@@ -0,0 +1,47 @@
+param(
+    [string]$searchDirectory = '.',
+    [hashtable]$filters = @{}
+)
+
+class StressTestPackageInfo {
+    [string]$Namespace
+    [string]$Directory
+    [string]$ReleaseName
+}
+
+function findStressPackages([string]$directory, [hashtable]$filters = @{}) {
+    # Bare minimum filter for stress tests
+    $filters['stressTest'] = 'true'
+
+    Get-ChildItem -Recurse -Filter 'Chart.yaml' -PipelineVariable chartFile $directory 
+      | % { parseChart $chartFile }
```

Comment:
While this might work in this context I've generally started to try avoiding `|` in scripts to avoid some of the weirdness that happens with them. I'd suggest capturing the output from Get-ChildItem and then doing a foreach look over that.

Also it is generally preferred to use full cmdlet names over alias in scripts (i.e. Foreach-Object over `%` and Where-Object over `?`)